### DISCUSSION TITLE
USB Drive can't be scheduled

### DISCUSSION THREAD

**Discussion by JohannesMerkt** [longhorn/longhorn#4943] [Q&A]
I have a raspberry pi running k3s with longhorn and I want to extend the storage capacity by adding a USB drive. 

the usb drive is exfat formatted as you can see here:
![grafik](https://user-images.githubusercontent.com/25324640/204142458-38076678-d265-448d-9e6a-0ba07940cd4f.png)

I've mounted the USB drive at /media/usb/ and created the folder longhorn inside. Using these steps:
![grafik](https://user-images.githubusercontent.com/25324640/204142237-22c9a37c-622a-4677-8045-7fad8b645142.png)

Ive added the drive in longhorn on the node:
![grafik](https://user-images.githubusercontent.com/25324640/204142266-5075fbdd-fa83-4428-8c4a-ea7aa68c4314.png)

and disabled the other drive for scheduling:
![grafik](https://user-images.githubusercontent.com/25324640/204142302-381e4045-9043-470b-80e2-6d7a68fb01a8.png)

when i then start a test pod using a storage class it gets stuck:
![grafik](https://user-images.githubusercontent.com/25324640/204142347-f54e852b-706a-42e5-89b2-8a28f3e500d6.png)

It creates something inside the drive though:
![grafik](https://user-images.githubusercontent.com/25324640/204142519-8f8676e9-bdcc-4454-b67a-794bfc98668b.png)

Why and how can i resolve it? Or how can I see why it failed?

**Comment by derekbit** [comment]
Can you check the disk conditions by `kubectl -n longhorn-system get nodes <node name> -o yaml`? Then, you can check the error message in the `status.diskStatus`.

**Reply by JohannesMerkt** [reply]
hmm there is no diskStatus
![grafik](https://user-images.githubusercontent.com/25324640/204142800-84d145df-c3b7-4e16-8782-1b2bd795b4cd.png)

**Comment by derekbit** [comment]
@JohannesMerkt 
Sorry, should use this command `kubectl -n longhorn-system get nodes.longhorn.io <node name> -o yaml`.

**Reply by JohannesMerkt** [reply]
sadly no error
![grafik](https://user-images.githubusercontent.com/25324640/204144198-f8136f85-97f9-4b3d-9ccb-eac38ff8bed2.png)

**Reply by JohannesMerkt** [reply]
Still tests keep failing
![grafik](https://user-images.githubusercontent.com/25324640/204144379-58731f11-c74c-480d-a4eb-095a94bb440c.png)

**Reply by derekbit** [reply]
OK. Can you check longhorn manager log or send up the support bundle? 
Ref: https://www.suse.com/support/kb/doc/?id=000020145

**Reply by JohannesMerkt** [reply]
Will do as soon as i have rancher up and running. Not entirely sure how to get longhorn manager logs. There are multiple longhorn manager pods. Sorry for my incompetence

**Reply by JohannesMerkt** [reply]
![grafik](https://user-images.githubusercontent.com/25324640/204417905-2a7f2ad4-d4e0-4a17-b993-e98d01a076a2.png)
the longhorn manager warns of mismatching disks

**Reply by derekbit** [reply]
OK. Can you check if the node or the disk is set to evicted? Looks your disk is set to evicted from the log.

**Reply by JohannesMerkt** [reply]
Not the node and disk with issues are eviction requested. Ive requested eviction on another disk to test if volumes can be moved to the disk with issues successfully but sadly no. Also to further test other solutions ive mounted a nfs share onto my system and tried to use it as disk, but this also fails.

**Reply by derekbit** [reply]
The infonmation in the [log](https://github.com/longhorn/longhorn/discussions/4943#discussioncomment-4258864) you provided is not matched with your description. You can send us the support bundle for futher analysis.
